documentation/src/main/wiki/FAQ.md

```diff
@@ -0,0 +1,144 @@
+# FAQ
+This section contains answers for frequently asked questions. If you can't find an answer for your question here
```

Comment:
_an answer **to** your question_

I think _to_ sounds a lot more natural in this context. Additionally: http://english.stackexchange.com/a/77300/24844